tests/scripts/run-corefx-tests.py

```diff
@@ -238,7 +238,7 @@ def main(args):
     log(command)
     returncode = 0 if testing else os.system(command)
     if not returncode == 0:
```

Comment:
Is there a functional difference between this "not returncode == 0" and the "return code != 0" used several places below?